test/unit/pdf_find_controller_spec.js

```diff
@@ -599,4 +605,29 @@ describe("pdf_find_controller", function () {
       ],
     });
   });
+
+  it("performs a search in a text containing some Hangul syllables", async function () {
+    if (isNodeJS) {
+      pending("Linked test-cases are not supported in Node.js.");
+    }
```

Comment:
Given that the test-case is now added directly to the repository, this shouldn't be needed now.